src/components/KonnectorSync.jsx

```diff
@@ -1,15 +1,26 @@
+import styles from '../styles/KonnectorSync'
+
 import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 import DescriptionContent from './DescriptionContent'
 
-export const KonnectorSync = ({ t, f, date }) => {
+export const KonnectorSync = ({ t, f, date, error, editing, submitting, onForceConnection }) => {
```

Comment:
See my comment below : `editing` is not needed anymore.
Maybe `error` neither ?